main/ZgatewayBT.ino

```diff
@@ -569,6 +569,8 @@ void BLEconnect() {
               if (pRemoteCharacteristic->canNotify()) {
                 Log.trace(F("Registering notification" CR));
                 pRemoteCharacteristic->subscribe(true, notifyCB);
+                delay(BLE_CNCT_TIMEOUT);
```

Comment:
Instead of a delay here we could use a task notification wait with BLE_CNCT_TIMEOUT as the max wait time. The notification callback could release the task before the timeout and this would save time and battery power for both devices if applicable. Thoughts?

Reply:
Yes good point!